projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -3,9 +3,7 @@
   class="cx-card"
   [tabindex]="border ? 0 : -1"
   [attr.role]="content.role ?? null"
-  [attr.aria-label]="
-    content.label ? (content.label | cxTranslate: { number: this.index }) : null
-  "
+  [attr.aria-labelledby]="ariaLabelledBy"
```

Comment:
Do we still need `ariaLabelledBy` if we have `getLabel`?

Reply:
I've just updated it.